src/int.rs

```diff
@@ -550,4 +628,38 @@ mod tests {
         assert!(zero < a);
         assert!(zero > b);
     }
+
+    #[test]
+    fn partialeq_test() {
+        let zero = Int::from(0);
+        let one = Int::from(1);
+        let negative_one = Int::from(-1);
+
+        assert!(zero == 0);
+        assert!(one == 1);
+        assert!(negative_one == -1);
```

Comment:
Add a case where this will fail. Right now `return true;` will pass these tests.